.github/workflows/push-dev-environment.yaml

```diff
@@ -51,36 +54,40 @@ jobs:
         env:
           # general
           NAMESPACE: eden-spending
-          INGRESS_HOST: spending.eden.eoscommunity.org
-          INGRESS_GRAPHQL_HOST: graphql-spending.eden.eoscommunity.org
+          INGRESS_HOST: spending.eden.edenia.cloud
+          INGRESS_GRAPHQL_HOST: spending-graphql.eden.edenia.cloud
+          IMAGE_NAME_WEBAPP: eden-spending-webapp-dev
+          IMAGE_NAME_HAPI: eden-spending-hapi-dev
+          IMAGE_NAME_HASURA: eden-spending-hasura-dev
+          IMAGE_NAME_WALLET: eden-spending-wallet-dev
           # Postgres
-          POSTGRES_USER: ${{ secrets.POSTGRES_USER }}
-          POSTGRES_PASSWORD: ${{ secrets.POSTGRES_PASSWORD }}
-          POSTGRES_DB: ${{ secrets.POSTGRES_DB }}
-          POSTGRES_DATA: ${{ secrets.POSTGRES_DATA }}
+          POSTGRES_USER: ${{ secrets.DEV_POSTGRES_USER }}
```

Comment:
we can use github environments to set different variables for each environment without having to change the env var names

Reply:
you right, i changed the variables and updated the values for de development environment secrets.